ocs_ci/ocs/resources/cloud_uls.py

```diff
@@ -48,6 +50,11 @@ def cloud_uls_factory(request, cld_mgr):
     except AttributeError:
         log.info("RGW is not available and was not initialized")
 
+    try:
+        ulsMap["aws-sts"] = cld_mgr.aws_sts_client
+    except Exception:
```

Comment:
Please use a more specific Exception here - I think that `AttributeError` is what we should expect here